nnpdf_data/nnpdf_data/new_commondata/LHCB_Z0_13TEV_2022/eigenvalues.yaml

```diff
@@ -0,0 +1,231 @@
+PT:
```

Comment:
Please put this file in the rawdata folder (that way it is easier to skip when distributing/installing the data)